docs/LICENSE_OF_DEPENDENCIES.md

```diff
@@ -490,7 +490,7 @@ following works:
 - gopkg.in/tomb.v2 [BSD 3-Clause Clear License](https://github.com/go-tomb/tomb/blob/v2/LICENSE)
 - gopkg.in/yaml.v2 [Apache License 2.0](https://github.com/go-yaml/yaml/blob/v2.2.2/LICENSE)
 - gopkg.in/yaml.v3 [MIT License](https://github.com/go-yaml/yaml/blob/v3/LICENSE)
-- howett.net/plist [BSD-2-Clause-Views, BSD-3-Clause](https://github.com/DHowett/go-plist/blob/main/LICENSE)
+- howett.net/plist [BSD 2-Clause with views sentence](https://github.com/DHowett/go-plist/blob/main/LICENSE)
```

Comment:
I don't think this is correct! The package actually has a mixed license so we can't just pretend it has another one for our tools to pass. ;-) Fixed in #17837.